Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Clickstream",
+    platforms: [.macOS(.v10_12),
+                .iOS(.v11),
+                .tvOS(.v10),
+                .watchOS(.v3)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "Clickstream",
+            targets: ["Clickstream"]),
+    ],
+    dependencies: [
+        .package(name: "SwiftProtobuf", url: "https://github.com/apple/swift-protobuf.git", from: "1.6.0"),
```

Comment:
Is 1.6.0 version of Swift-ProtoBuf?

In [Podfile](https://github.com/gojek/clickstream-ios/blob/98d2368c61fd64c9a9c2591241fc31767283298b/Podfile#L12) we are using 1.10.2? Will that cause discrepancy?

Reply:
In Package.resolved the "version": "1.19.0" is mentioned

Reply:
Yes I updated it to be of version 1.10.2, similar to pod